public/app/core/components/PageHeader/PageHeader.tsx

```diff
@@ -84,7 +83,7 @@ export const PageHeader: FC<Props> = ({ model, contentWidth }) => {
 
   return (
     <div className={styles.headerCanvas}>
-      <div className={cx('page-container', contentWidth ? styles.contentWidth(contentWidth) : undefined)}>
+      <div className={cx('page-container')}>
```

Comment:
Nit: you should be able to skip the `cx` in this scenario.

Reply:
- [ ] remove cx